### PR TITLE
feat: add new language (lt-LT)

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+"Language: lt-LT\n"
+"sourceLanguage: en-US\n"
+"targetLanguage: lt-LT\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-Basepath: ../../\n"
+"X-Poedit-KeywordsList: __\n"
+"X-Poedit-SearchPath-0: .\n"
+
+msgid " . \common_Utils::toPHPVariableString($this->getDescription()) . "
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: Are you sure that we need to translate it?

File: locales/lt-LT/messages.po [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+msgstr "Čia taip pat konfigūruojami testo režimas (klausimų eiliškumas, vertinimas, taškų sumavimas) ir testo išdėstymas."
+
+msgid "The Tests module allows to combine a selection of Items to a Test."
+msgstr "Testo modulis leidžia apjungti pasirinktus Elementus į Testą."
+
+msgid "Unable to delete test."
+msgstr ""
+
+msgid "Unable to process your request"
+msgstr ""
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: Do we need to translate it?

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: no, some of them might be empty

File: locales/lt-LT/messages.po (hunk 1 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

Comment (bartlomiejmarszal) [reply locales/lt-LT/messages.po]: How is it fixed if we still se this?

**Pull request merged**